sdk/eventhubs/azure_messaging_eventhubs/tests/consumer.rs

```diff
@@ -164,6 +164,7 @@ async fn receive_lots_of_events() -> Result<(), Box<dyn Error>> {
         .open(host.as_str(), eventhub.as_str(), credential.clone())
         .await?;
 
+    info!("Client open, create receiver.");
```

Comment:
Per previous comment, this doesn't doesn't seem to rise to the level of `info`. FWIW, after talking with Catalina we'll probably end up tracing methods as `info`, stuff most developers might want to see as `debug`, and really verbose stuff as `trace`.

Reply:
This is test code.

For my tests, I use "info" as a shorthand for `println` - but it's controlled by the RUST_LOG environment variable.